packages/mongodb-chatbot-server/src/routes/conversations/addMessageToConversation.ts

```diff
@@ -312,11 +312,29 @@ async function sendStaticNonResponse({
   );
   const apiRes = convertMessageFromDbToApi(assistantMessage);
   if (shouldStream) {
+    const { metadata, references } = apiRes;
+
     dataStreamer.connect(res);
+
+    if (metadata) {
+      dataStreamer.streamData({
+        type: "metadata",
+        data: metadata,
+      });
+    }
```

Comment:
I think currently we'll only stream a `metadata` event for verified answers, right? Do we need any conditional handling for delta streaming (immediately below this block) in the case of verified answers?

Or, is the idea that the `metadata` event is used solely to indicate that any delta events until the next `finished` event  is a verified answer?

I ask because it looks like the `metadata` payload will include the verified answer text in its entirety so we don't _technically_ need to stream deltas on top of it.